src/tests/functional/shared_test_classes/src/single_op/stft.cpp

```diff
@@ -0,0 +1,91 @@
+// Copyright (C) 2018-2024 Intel Corporation
```

Comment:
Please move this file to https://github.com/openvinotoolkit/openvino/tree/master/src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/classes